README.md

```
@@ -12,25 +12,205 @@
+
+Run mutation testing:
+
+```bash
+./gradlew pitest
+```
+
+Coverage is enforced at ≥ 80% via Kover and ≥ 50% via Jacoco verification.
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)